### ISSUE TITLE
Multi Monitor Failure

### ISSUE THREAD

**Issue by Coachmn** [marijnkampf/Random-Photo-Screensaver#119]
On RPS 3.4 and upgraded to the latest 4.5 version. However unable to make the multi monitor work. Whatever the setting, one photos is shown spanning 3 monitors. On occasions it will correctly show a photo on monitor 1 but shows a part photo on monitors 2 and 3.
PC Packard Bell ixtreme M5722 6GB RAM
Processor Intel Core2 Quad CPU Q8300 @ 2.5GHz
Graphics 1GB NVIDIA GeForce GTX650 Ti (Driver 25.21.14.1634)
Windows 10 Home 10.0.16299 Build 16299

**Comment by marijnkampf** [comment]
It sounds what you are describing are the result of some new features of RPS. Please try the following in Settings (Press S to enter whilst RPS is running):

Slideshow > Effects > **Disable** Zoom and Pan / Ken Burns Effect
Monitors > Multi monitors > **Un-check** Stretch panoramas across multiple monitors

**Comment by marijnkampf** [comment]
Can you please double check that:

Multi monitors > Stretch panoramas across multiple monitors is not checked?

Please note that a photograph is treated as a panorama as if the width / height ratio of the panorama is equal or greater than the width / height ratio of the collection of monitors.

For example a 2800x900 panorama won't be displayed if your actual screen size is 2880x900.

Only panorama's selected on monitor 1 are stretched. If monitor 1 has a regular photo and monitor 2 a panorama, the panorama won't get stretched because of the normal picture on monitor 1.

If this doesn't resolve it could you sent me your configuration file for me to investigate further. You can find them For Windows Vista, 7, 8/8.1, 10:

`Go to the folder %programdata%\Random Photo Screensaver\`
`Locate: settings.sqlite`

**Comment by Coachmn** [comment]
I previously loaded and tried v4.5 about 3 times and had, as this time, reverted to v3.4. Today I loaded v4.5 again and from the installer went directly to the settings. Either v4.5 picked up the config file from v3.4 or the v4.5 config file had not been deleted when I removed the v4.5 last time.

Whatever the reason, the settings were already set to my preferences. Having checked the Stretch Panoramas was not set, I went into a trial. It took about 10 minutes and had scanned about 75,000 files when it terminated prematurely (possibly because my e-mail client downloaded a mail?). I started RPS again and watched for about another 5 minutes. In all this time it performed perfectly unlike the last 3 times it had been tried.

It seems therefore that RPS is now functioning correctly. Could it be that when I have deselected Stretch Panoramas previously the config file has not been reloaded without a re-boot and consequently never been implemented because after failure I've just removed v4.5 and never tried it from a fresh boot?
